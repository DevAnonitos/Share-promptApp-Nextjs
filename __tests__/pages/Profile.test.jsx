import React from "react";
import MyProfile from "@/app/profile/page";
import UserProfile from "@/app/profile/[id]/page";
import { render, screen } from "@testing-library/react";

describe("Profile Page", () => {
  it("should render MyProfile Page Component", () => {
    expect(MyProfile).toBeDefined()
    expect(MyProfile).toMatchSnapshot()
  })

  it("should render UserProfile Page Component", () => {
    expect(UserProfile).toBeDefined()
    expect(UserProfile).toMatchSnapshot()
  })
})