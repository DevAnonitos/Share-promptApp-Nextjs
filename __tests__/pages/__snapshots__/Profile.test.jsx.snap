// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Page should render MyProfile Page Component 1`] = `[Function]`;

exports[`Profile Page should render UserProfile Page Component 1`] = `[Function]`;
